lib/renderer/security-warnings.ts

```diff
@@ -1,6 +1,14 @@
-'use strict'
+import { webFrame } from 'electron'
+import { invokeSync } from '@electron/internal/renderer/ipc-renderer-internal-utils'
 
-let shouldLog = null
+declare global {
```

Comment:
Can this be put in our `ambient-*.d.ts` files so we don't have type mutations in individual files?

Reply:
You can't mutate the global namespace from non-module files, so sadly not.

Reply:
@felixrieseberg You can mutate the top level `Window` interface though

```
declare interface Window {
  hideLoader: () => void;
}
```

(copy pasted from another project)